docs/toc.json

```diff
@@ -71,6 +71,9 @@
       "title": "Region",
       "type": "compute/region"
     }, {
+      "title": "Service",
+      "type": "compute/service"
+    }, {
```

Comment:
I like it! My local "deploy preview docs" script was wrong. It's in the sidebar in the preview now: http://stephenplusplus.github.io/gcloud-node/#/docs/master/compute/service